src/locales/pt_BR/battle.ts

```diff
@@ -13,7 +13,7 @@ export const battle: SimpleTranslationEntries = {
   "trainerGo": "{{trainerName}} escolheu {{pokemonName}}!",
   "switchQuestion": "Quer trocar\nde {{pokemonName}}?",
   "trainerDefeated": "Você derrotou\n{{trainerName}}!",
-  "moneyWon": "Você ganhou\n₽{{moneyAmount}} por vencer!",
+  "moneyWon": "Você ganhou\n₽{{moneyAmount}} por ganhar!",
```

Comment:
```suggestion
  "moneyWon": "Você ganhou\n₽{{moneyAmount}} por vencer!",
```

The correct word is "vencer" and it has been fixed earlier